### PR TITLE
Update PULP_CONTENT_HOST to PULP_CONTENT_ORIGIN

### PR DESCRIPTION
**Pull request by alikins**
This is based on:
  https://github.com/pulp/pulp_ansible/commit/e8a9176cef731b7646aa770608ec2276802ccd10

which is based on pulpcore change:
  https://github.com/pulp/pulpcore/pull/370

which implements:
  https://pulp.plan.io/issues/5629

### CODE REVIEW

**Comment by alikins** [comment]
@bmbouter, @daviddavis Mind taking a look

**Comment by bmbouter** [comment]
`CONTENT_ORIGIN` needs to have a scheme, host, and port in it. It's more than a name change. See the docs here:  https://github.com/pulp/pulpcore/blob/master/docs/installation/configuration.rst#content_origin

When setting it you put `PULP_` on the front because that's how dynaconf layers settings onto Pulp. Docs on that are here: https://github.com/pulp/pulpcore/blob/master/docs/installation/configuration.rst#by-environment-variables

**Comment by bmbouter** [comment]
Also I believe @cutwater already applied a change like this somewhere?

**Comment by bmbouter** [comment]
Oh actually I think that was in the other repo, not the galaxy-api repo. Perhaps that's where the other change occurred.

**Comment by alikins** [comment]
> Also I believe @cutwater already applied a change like this somewhere?

Hmm, maybe, but I don't see anything in origin/master to indicate that.

**Comment by bmbouter** [comment]
@alikins I agree I also don't see it. I had heard about if from slack so I'm not sure.

**Comment by cutwater** [comment]
It's applied for production config. For development this change is still needed.

**Comment by cutwater** [comment]
But this is not the right place to apply this change. It's a config for galaxy, not pulp.
`PULP_CONTENT_HOST` and `PULP_CONTENT_PORT` parameters are not related to pulp configuration. These indicate connection port and host for galaxy.

You should update `docker/pulp/settings.py` and set `CONTENT_ORIGIN` there.

**Comment by cutwater** [comment]
PR that fixes this issue:
https://github.com/ansible/galaxy-dev/pull/171

**Comment by bmbouter** [comment]
@cutwater you likely correct about this because I haven't worked with the galaxy code closely like you have. FYI, prepending pulp options with `PULP_` is how to [set a Pulp setting via environment variable](https://docs.pulpproject.org/en/3.0/nightly/installation/configuration.html#by-environment-variables). I wanted to share some information in case it's helpful.

**Comment by cutwater** [comment]
@bmbouter I'm aware of PULP_ prefix usage in environment variables.
We use `GALAXY_` prefix for `galaxy-api` service, similar to how pulp is configured.

This configuration is part of `galaxy-api` configuration and has no direct relation to pulp services.

File: galaxy_api/settings.py [hunk 1]
```diff
@@ -130,7 +130,7 @@
 PULP_API_USER = 'admin'
 PULP_API_PASSWORD = 'admin'
 
-PULP_CONTENT_HOST = 'pulp-content-app'
+PULP_CONTENT_ORIGIN = 'pulp-content-app'
```
Comment (alikins) [inline galaxy_api/settings.py]: Needs to be full url (ie, https://cloud.redhat.com/api/automation-hub/ etc)

**Pull request closed without merging**